message/validation/event_validation.go

```diff
@@ -0,0 +1,19 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/bloxapp/ssv/protocol/v2/ssv/queue"
+	ssvtypes "github.com/bloxapp/ssv/protocol/v2/types"
+)
+
+// event_validation.go contains methods for validating event messages
+
+func (mv *MessageValidator) validateEventMessage(msg *queue.DecodedSSVMessage) error {
+	_, ok := msg.Body.(*ssvtypes.EventMsg)
+	if !ok {
+		return fmt.Errorf("expected event message")
+	}
+
+	return ErrEventMessage
+}
```

Comment:
events are internal messages (such as round timeout or execute duty) and aren't supposed to be sent by peers
so i think we can remove this and just return an error when receiving an event message as early as possible in the validation pipeline